sdcm/stress_thread.py

```diff
@@ -158,11 +158,23 @@ def _run_stress(self, node, loader_idx, cpu_idx, keyspace_idx):
                                      cs_log_filename=log_file_name,
                                      loader_idx=loader_idx, cpu_idx=cpu_idx), \
                 CassandraStressEventsPublisher(node=node, cs_log_filename=log_file_name):
-
-            result = node.remoter.run(cmd=node_cmd,
-                                      timeout=self.timeout,
-                                      ignore_status=True,
-                                      log_file=log_file_name)
+            result = None
+            try:
+                result = node.remoter.run(cmd=node_cmd,
+                                          timeout=self.timeout,
+                                          ignore_status=True,
+                                          log_file=log_file_name)
+            except Exception as exc:  # pylint: disable=broad-except
+                CassandraStressEvent(type='failed', node=str(node), stress_cmd=stress_cmd,
```

Comment:
better `type='failure'`
since start, finish are nouns

Reply:
Won't it be enough just to change the `remoter.run()` to use `ignore_status=False` ?
And then we could only catch exceptions, and not have this duplication of events sending code